src/pages/ScreenRouting/MediaScreenRouter.tsx

```diff
@@ -8,25 +8,26 @@ import useMedia from '#src/hooks/useMedia';
 import Loading from '#src/pages/Loading/Loading';
 import ErrorPage from '#components/ErrorPage/ErrorPage';
 import type { PlaylistItem } from '#types/playlist';
-import { isEpisode, isSeriesPlaceholder } from '#src/utils/media';
+import { isEpisode, isDeprecatedSeriesFlow } from '#src/utils/media';
 import MediaMovie from '#src/pages/ScreenRouting/mediaScreens/MediaMovie/MediaMovie';
 import MediaSeries from '#src/pages/ScreenRouting/mediaScreens/MediaSeries/MediaSeries';
 import MediaSeriesEpisode from '#src/pages/ScreenRouting/mediaScreens/MediaSeriesEpisode/MediaSeriesEpisode';
 import MediaLiveChannel from '#src/pages/ScreenRouting/mediaScreens/MediaLiveChannel/MediaLiveChannel';
 import { ScreenMap } from '#src/pages/ScreenRouting/ScreenMap';
+import { CONTENT_TYPE } from '#src/config';
 
 export const mediaScreenMap = new ScreenMap<PlaylistItem>();
 
-// register media screens
-mediaScreenMap.registerByContentType(MediaSeries, 'series');
-mediaScreenMap.registerByContentType(MediaSeriesEpisode, 'episode');
-mediaScreenMap.registerByContentType(MediaLiveChannel, 'livechannel');
-mediaScreenMap.registerByContentType(MediaStaticPage, 'page');
+// Register media screens
+mediaScreenMap.registerByContentType(MediaSeries, CONTENT_TYPE.series);
+mediaScreenMap.registerByContentType(MediaSeriesEpisode, CONTENT_TYPE.episode);
+mediaScreenMap.registerByContentType(MediaLiveChannel, CONTENT_TYPE.livechannel);
+mediaScreenMap.registerByContentType(MediaStaticPage, CONTENT_TYPE.page);
 mediaScreenMap.registerDefault(MediaMovie);
 
-// register legacy series and episode screens when `contentType` is missing
+// Register legacy series and episode screens when `contentType` is missing
 mediaScreenMap.register(MediaSeriesEpisode, (item) => !!item && isEpisode(item));
```

Comment:
So because Marco doesn't like the /s/ route for anything but the deprecated flow, let's map these 2 items and the /s/ path to a legacy series screen where we can isolate as many of the differences between old and new flow as possible.